src/pages/Beta/Pool/index.tsx

```diff
@@ -155,6 +168,22 @@ const PoolUI = () => {
     [setMenu]
   )
 
+  const getVersion = () => {
+    const chefType = minichef?.type
+    switch (chefType) {
+      case ChefType.MINI_CHEF:
+        return 1
+      case ChefType.MINI_CHEF_V2:
+        return 2
+      case ChefType.PANGO_CHEF:
+        return 3
+      default:
+        return 2
+    }
+  }
```

Comment:
we don't want to use numbers for version. only want to use ChefType, otherwise there is no meaning.